--- application/tmp/webister/interface/cloudflare.php
+++ PHP_CodeSniffer
@@ -1,15 +1,15 @@
 <?php session_start(); require 'include/head.php'; 
-require ("include/cloudflare.php");
-require ("include/mail.php");
+require "include/cloudflare.php";
+require "include/mail.php";
 if (isset($_POST["email"])) {
-    $uniqueid = rand(1,10000);
+    $uniqueid = rand(1, 10000);
     $usernameit = $uniqueid . $_SESSION['user'];
     $email = $_POST["email"];
     $password = $_POST["password"];
     $cf = new cloudflare_api(file_get_contents("data/cloudflare"));
-$response = $cf->user_create($email, $password, $usernameit, $uniqueid);
-sendemailuser("New Cloudflare","A new cloudflare account has been created!!<br>Username: " . $usernameit . "<br>Password:" . $password . "<br>Email: " . $email . "<br>Unique Id: " . $uniqueid . "<br><p>Thank You for using the Cloudflare API</p>");
-$mysqli = new mysqli();
+    $response = $cf->user_create($email, $password, $usernameit, $uniqueid);
+    sendemailuser("New Cloudflare", "A new cloudflare account has been created!!<br>Username: " . $usernameit . "<br>Password:" . $password . "<br>Email: " . $email . "<br>Unique Id: " . $uniqueid . "<br><p>Thank You for using the Cloudflare API</p>");
+    $mysqli = new mysqli();
         $conn = mysqli_connect("$host", "$user", "$pass", "$data"); 
          $sql = "INSERT INTO Cloudflare (id, username, email, password)
 VALUES ('" .$uniqueid ."','" .$_SESSION['user'] . "','" .$email ."','" . $password."')";
--- application/tmp/webister/interface/cp.php
+++ PHP_CodeSniffer
@@ -7,13 +7,13 @@
   
 </style>
             <?php if ($_SESSION['user'] == 'admin') {
-$v = file_get_contents("data/version");
-$c = file_get_contents("http://mirror.adaclare.com/updates/current");
-if ($v = "XXX" || $c > $v) {
-    ?>
-    <div class="alert alert-danger">A new update is avaliable, Please visit the update center to install the update.</div>
-    <?php
-}
+                $v = file_get_contents("data/version");
+                $c = file_get_contents("http://mirror.adaclare.com/updates/current");
+                if ($v = "XXX" || $c > $v) {
+                    ?>
+                    <div class="alert alert-danger">A new update is avaliable, Please visit the update center to install the update.</div>
+                    <?php
+                }
 ?>
 
                       <div data-step="1" data-intro="Welcome to Webister Platform, this is where you can see info about different versions." style="   margin-top: 5px;
@@ -23,18 +23,18 @@
   <h1>News</h1>
   <p><?php 
   
-  $news = file_get_contents("http://mirror.adaclare.com/messages/NEW.txt"); 
-  if (strlen($news) > "190") {
-    echo substr($news, 0, 190) . "...</p>";
-    ?>
+    $news = file_get_contents("http://mirror.adaclare.com/messages/NEW.txt"); 
+    if (strlen($news) > "190") {
+        echo substr($news, 0, 190) . "...</p>";
+        ?>
     
-    <p><a class="btn btn-primary btn-lg" href="http://mirror.adaclare.com/messages/NEW.txt" role="button">Learn more</a><a class="btn btn-primary btn-lg" href="javascript:void(0);" onclick="javascript:introJs().start();" role="button">Take The Tour</a></p>
-    <?php
-  } else {
-   echo $news . "</p>";
-  }
+        <p><a class="btn btn-primary btn-lg" href="http://mirror.adaclare.com/messages/NEW.txt" role="button">Learn more</a><a class="btn btn-primary btn-lg" href="javascript:void(0);" onclick="javascript:introJs().start();" role="button">Take The Tour</a></p>
+        <?php
+    } else {
+        echo $news . "</p>";
+    }
   
-  ?>
+    ?>
   
 </div>  
 <div>
@@ -57,51 +57,51 @@
   <li class="list-group-item"><a id="baahide"><i class="fa fa-list" aria-hidden="true"></i></a><a id="baashow"><i class="fa fa-list" aria-hidden="true"></i></a> | Status</li>
   <li class="list-group-item">  
                                           <a type="button" href="" class="aasvr btn btn-default"><h1 style="font-size: 60px;"><?php
-    $count = 0;
-$con = mysqli_connect($host, $user, $pass, $data);
-$sql = 'SELECT * FROM Users';
-$result = mysqli_query($con, $sql);
- while ($row = mysqli_fetch_row($result)) {
-     $count = $count + 1;
- }
-   mysqli_free_result($result);
-    mysqli_close($con);
-    echo $count;
+                                            $count = 0;
+                                            $con = mysqli_connect($host, $user, $pass, $data);
+                                            $sql = 'SELECT * FROM Users';
+                                            $result = mysqli_query($con, $sql);
+                                            while ($row = mysqli_fetch_row($result)) {
+                                                $count = $count + 1;
+                                            }
+                                            mysqli_free_result($result);
+                                            mysqli_close($con);
+                                            echo $count;
     ?></h1><hr> Users</a>
                                           <a type="button" href="" class="aasvr btn btn-default"><h1 style="font-size: 60px;"><?php
-    $count = 0;
-$con = mysqli_connect($host, $user, $pass, $data);
-$sql = 'SELECT * FROM FailedLogin';
-$result = mysqli_query($con, $sql);
- while ($row = mysqli_fetch_row($result)) {
-     $count = $count + 1;
- }
-   mysqli_free_result($result);
-    mysqli_close($con);
-    echo $count;
+                                            $count = 0;
+                                            $con = mysqli_connect($host, $user, $pass, $data);
+                                            $sql = 'SELECT * FROM FailedLogin';
+                                            $result = mysqli_query($con, $sql);
+                                            while ($row = mysqli_fetch_row($result)) {
+                                                $count = $count + 1;
+                                            }
+                                            mysqli_free_result($result);
+                                            mysqli_close($con);
+                                            echo $count;
     ?></h1><hr> Failed Logins</a>
                                                                                 <a type="button" href="" class="aasvr btn btn-default"><h1 style="font-size: 60px;"><?php
-    $count = 0;
-$con = mysqli_connect($host, $user, $pass, $data);
-$sql = 'SELECT * FROM Users';
-$result = mysqli_query($con, $sql);
- while ($row = mysqli_fetch_row($result)) {
-     $count = $count + 1;
- }
-   mysqli_free_result($result);
-    mysqli_close($con);
-    echo $count;
+                                                                                $count = 0;
+                                                                                $con = mysqli_connect($host, $user, $pass, $data);
+                                                                                $sql = 'SELECT * FROM Users';
+                                                                                $result = mysqli_query($con, $sql);
+                                                                                while ($row = mysqli_fetch_row($result)) {
+                                                                                    $count = $count + 1;
+                                                                                }
+                                                                                mysqli_free_result($result);
+                                                                                mysqli_close($con);
+                                                                                echo $count;
     ?></h1><hr> Servers</a>
                                                                                                                         <a type="button" href="" class="aasvr btn btn-default"><h1 style="font-size: 60px;">
-                                                                                                                          <?php
-                                                                                                                          $count = 0;
-                                                                                                                          $scan = scandir("plugins");
-                                                                                                                          foreach ($scan as $file) {
-                                                                                                                            $count = $count +1;
-                                                                                                                          }
-                                                                                                                          $count = $count - 2;
-                                                                                                                          echo $count;
-                                                                                                                          ?>
+                                                                                                                            <?php
+                                                                                                                            $count = 0;
+                                                                                                                            $scan = scandir("plugins");
+                                                                                                                            foreach ($scan as $file) {
+                                                                                                                                $count = $count +1;
+                                                                                                                            }
+                                                                                                                            $count = $count - 2;
+                                                                                                                            echo $count;
+                                                                                                                            ?>
                                                                                                                           
                                                                                                                         </h1><hr> Plugins</a>
                                           <a type="button" href="" class="aasvr btn btn-default"><h1 style="font-size: 60px;"><?php echo file_get_contents("data/version"); ?></h1><hr> Version</a>
@@ -157,7 +157,7 @@
                                             ?>
                                         
                                           <a type="button" href="cloudflare.php" class="svr btn btn-default"><i class="fa fa-5x fa-cloud"></i><hr>Cloudflare</a>
-                                          <?php } ?>
+                                        <?php } ?>
                                         <a type="button" href="index.php?page=list#" class="svr btn btn-default"><i class="fa fa-5x fa-user"></i><hr>Users</a>
                                         <a type="button" href="plans.php" class="svr btn btn-default"><i class="fa fa-5x fa-columns" aria-hidden="true"></i><hr>Plans</a>
                                         <a type="button" href="adminer-4.2.4.php?server=localhost&username=<?php echo urlencode($user); ?>&password=<?php echo urlencode($pass); ?>" class="svr btn btn-default"><i class="fa fa-5x fa-database" aria-hidden="true"></i><hr> All Database</a>
@@ -191,21 +191,21 @@
                                       
                                         <a type="button" href="http://adaclare.com/errtrck/bug_report_page.php" class="sys btn btn-default"><i class="fa fa-5x fa-life-ring"></i><hr>Bugs</a>
                                         <a type="button" class="sys btn btn-large btn-success" href="javascript:void(0);" onclick="javascript:introJs().start();" class="btn btn-default"><i class="fa fa-5x fa-question-circle"></i><hr>Tour</a>
-                                       <?php
-                                       $scan = scandir("plugins/");
-foreach ($scan as $file) {
-    include("plugins/" . $file);
-    if ($menu) {
-       echo '<a type="button" class="sys btn btn-large btn-default" href="plpage.php?pl=' . urlencode($file) . '" class="btn btn-default"><i class="fa fa-5x fa-puzzle-piece"></i><hr>' . $menu_name . '</a>';
-    }
-}
-                                       ?>
+                                        <?php
+                                        $scan = scandir("plugins/");
+                                        foreach ($scan as $file) {
+                                            include "plugins/" . $file;
+                                            if ($menu) {
+                                                echo '<a type="button" class="sys btn btn-large btn-default" href="plpage.php?pl=' . urlencode($file) . '" class="btn btn-default"><i class="fa fa-5x fa-puzzle-piece"></i><hr>' . $menu_name . '</a>';
+                                            }
+                                        }
+                                        ?>
                                        
                                          </li>
   </ul>  
                                
             <?php 
-} ?>
+            } ?>
 
 <br>
 <script>
@@ -262,7 +262,7 @@
 </div>
 <?php
 if (!file_exists("data/lock")) { 
-  file_put_contents("data/lock","data/lock");
+    file_put_contents("data/lock", "data/lock");
 ?>
          <script type="text/javascript">
     $(window).load(function(){
--- application/tmp/webister/interface/main.php
+++ PHP_CodeSniffer
@@ -475,11 +475,11 @@
       <div class="button-container">
         <button><span>Go</span></button>
       </div>
-      	<?php
-			if (isset($_GET["error"])) {
-				echo "<div class='footer'>Wrong Username or Password!</div>";
-			}
-			?>
+        <?php
+        if (isset($_GET["error"])) {
+              echo "<div class='footer'>Wrong Username or Password!</div>";
+        }
+    ?>
 
     </form>
   </div>
--- application/tmp/webister/interface/newserv.php
+++ PHP_CodeSniffer
@@ -136,12 +136,14 @@
         header('Location: ?pa='.urlencode($returnval));
     }
     newserv($_POST['pstart'], $_POST['disk'], $_POST['username'], $_POST['pend']);
-       include("include/mail.php");
-            sendemailuser("New User","
+       include "include/mail.php";
+            sendemailuser(
+                "New User", "
     <b>A new user has been added to Webister</b>
     <p>There username is " . $_POST['username'] . "</p>
     <p>This email is automatically sent out everytime a setting is changed. To disable this feature please visit the control panel and set the email to nothing.</p>
-    ");
+    "
+            );
 }
 require 'include/head.php';
 onlyadmin();
--- application/tmp/webister/interface/pass.php
+++ PHP_CodeSniffer
@@ -16,9 +16,9 @@
    $query = sprintf("SET PASSWORD FOR %s@localhost = PASSWORD('%s');", $_POST['username'], $_POST['password']);
     mysqli_query($con, $query);
     mysqli_close($con);
-    if ($_POST["username"] == "admin" || $_POST["username"] == "Admin") {
-        include("include/mail.php");
-        sendemailuser("Password Changed","The password for the main administrator account has been changed. If you did not change your password we recommend that you restore the password or reinstall webister. This may also be an issue with the installation of project webister.");
-    }
+if ($_POST["username"] == "admin" || $_POST["username"] == "Admin") {
+    include "include/mail.php";
+    sendemailuser("Password Changed", "The password for the main administrator account has been changed. If you did not change your password we recommend that you restore the password or reinstall webister. This may also be an issue with the installation of project webister.");
+}
     
     header('Location: index.php?page=cp#');
--- application/tmp/webister/interface/temppass.php
+++ PHP_CodeSniffer
@@ -11,11 +11,11 @@
 <!doctype html>
 <html>
 <head>
-	<meta charset="UTF-8">
+    <meta charset="UTF-8">
     <link rel="shortcut icon" type="image/x-icon" href="images/favicon.ico">
     <title>Webister</title>
     <meta content='width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no' name='viewport'>
-	<link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet" integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN" crossorigin="anonymous">
+    <link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet" integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN" crossorigin="anonymous">
 <style>
 body {
   background: #e9e9e9;
@@ -460,8 +460,8 @@
 </head>
 <body>
 <div class="container">
-	<div class="row">
-		
+    <div class="row">
+        
 <!-- Mixins-->
 <!-- Pen Title-->
 <div class="pen-title">
--- application/tmp/webister/interface/title.php
+++ PHP_CodeSniffer
@@ -26,8 +26,10 @@
     file_put_contents('data/cloudflare', $_POST['cloudflare']);
     file_put_contents('data/upbutton', $_POST['upgrade']);
         file_put_contents('data/mail', $_POST['mail']);
-    sendemailuser("Settings Changed","
+    sendemailuser(
+        "Settings Changed", "
     <b>Settings have been changed on the Webister Hosting Control Panel</b>
     <p>This email is automatically sent out everytime a setting is changed. To disable this feature please visit the control panel and set the email to nothing.
-    ");
+    "
+    );
     header('Location: index.php?page=cp#');

